malariagen_data/anoph/h12.py

```diff
@@ -297,10 +306,10 @@ def h12_gwss(
         random_seed: base_params.random_seed = 42,
         chunks: base_params.chunks = base_params.native_chunks,
         inline_array: base_params.inline_array = base_params.inline_array_default,
-    ) -> Tuple[np.ndarray, np.ndarray]:
+    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
         # Change this name if you ever change the behaviour of this function, to
         # invalidate any previously cached data.
-        name = "h12_gwss_v1"
+        name = "h12_gwss_contig_v1"
```

Comment:
```suggestion
        name = "h12_gwss_v2"
```

Nit, convention is to use the function name then add a version number.